CarShowRoom/cars/api/v1/views.py

```diff
@@ -0,0 +1,61 @@
+from django.core.exceptions import ObjectDoesNotExist
+from rest_framework import status
+from rest_framework.mixins import (
+    CreateModelMixin,
+    ListModelMixin,
+    RetrieveModelMixin,
+    UpdateModelMixin,
+)
+from rest_framework.response import Response
+from rest_framework.viewsets import GenericViewSet
+
+from cars.api.serializers import CarBrandSerializer, CarSerializer
+from cars.models import Car, CarBrand
+
+
+class CarBrandViewSet(
+    ListModelMixin,
+    UpdateModelMixin,
+    RetrieveModelMixin,
+    CreateModelMixin,
+    GenericViewSet,
+):
+    """ViewSet for retrieving, updating, creating and getting all CarBrand instances"""
+
+    queryset = CarBrand.objects.filter(is_active=True)
+    serializer_class = CarBrandSerializer
+    lookup_field = "slug"
+    lookup_url_kwarg = "slug"
+
+
+class CarViewSet(
+    ListModelMixin,
+    RetrieveModelMixin,
+    CreateModelMixin,
+    UpdateModelMixin,
+    GenericViewSet,
+):
+    """ViewSet for creating, retrieving, updating, getting all Car instances"""
+
+    queryset = Car.objects.all()
```

Comment:
Should we filter is active here as well?

Reply:
agree, will fix